src/redux/features/AuthSlice/EditorSlice.ts

```diff
@@ -45,23 +79,43 @@ export const editorSlice = createSlice({
     setQuery: (state, action: PayloadAction<string>) => {
       state.query = action.payload
     },
+    setVariables: (state, action: PayloadAction<string>) => {
+      state.variables = action.payload
+    },
+    setHeaders: (state, action: PayloadAction<string>) => {
+      state.headers = action.payload
+    },
   },
 
   extraReducers: (builder) => {
-    builder.addCase(hydrate, (state, action) => {
-      return {
-        ...state,
-        ...action.payload.editorSlice,
-      }
-    }),
-      builder.addCase(getGqlValueThunk.fulfilled, (state, action) => {
+    builder
+      .addCase(hydrate, (state, action) => {
+        return {
+          ...state,
+          ...action.payload.editorSlice,
+        }
+      })
+      .addCase(getGqlValueThunk.pending, (state) => {
+        state.status = 'loading'
+      })
+      .addCase(getGqlValueThunk.fulfilled, (state, action) => {
         state.data = action.payload
+        console.log(action.payload)
+        state.status = 'idle'
+      })
+      .addCase(getGqlValueThunk.rejected, (state, action) => {
+        state.status = 'idle'
```

Comment:
maybe create global const or enum with values, cause you compare with it in component

Reply:
will be fixed/refactor in following commits